### ISSUE TITLE
Feature Request: OpenKJ Songbook - Request Apps to give successful message

### ISSUE THREAD

**Issue by backstagekaraoke** [OpenKJ/OpenKJ#158]
OpenKJ Songbook App
iOS, Android

At the moment when you submit a song, it just goes back to the main search screen, it doesn't give a 'Your song was submitted successfully' (or similar) message, so the patrons are sometimes unsure, and either submit it again (starting the same cycle) or come constantly ask.

Little thing, but could cause a bit of unnecessary drama.

**Comment by backstagekaraoke** [comment]
@coyote1357 I noticed that on the Android version, it now gives this successfully submitted message .... did you add this?

**Comment by coyote1357** [comment]
Yeah, it was added.  Will be added on iOS as well as soon as I test and tweak the current android version to work well on iOS and get working builds to submit to Apple.
